.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,4 @@
+- [ ] `uglify-js` version (`uglifyjs -V`)
+- [ ] Syntactically correct self-contained JavaScript input - ideally as small as possible.
```

Comment:
Ah. I misunderstood you the first time 😅

I thought you meant `javascript` ➡️ `JavaScript` for some weird reason.